components/combobox/index.css

```diff
@@ -377,6 +405,22 @@
 		&.is-invalid .spectrum-Textfield-validationIcon {
 			inset-inline-end: var(--mod-combobox-button-width, var(--spectrum-combobox-button-width));
 		}
+
+		&.spectrum-Textfield--quiet {
+			input:read-only {
```

Comment:
Similar comment here: if it was me, I'd refactor `input:read-only` to work with `.spectrum-Combobox-input:read-only` instead.